packages/block-editor/src/components/global-styles/border-panel.js

```diff
@@ -298,7 +298,7 @@ export default function BorderPanel( {
 						</BaseControl.VisualLabel>
 					) : null }
 
-					<ItemGroup isBordered isSeparated>
+					<ItemGroup isBordered isSeparated role="none">
```

Comment:
I suggest to add an inline comment `// TODO: ` to explain why this role none is used here, as a temporary fix. There are a few examples of todo comments in the codebase for inspiration.